kratos/utilities/qef_utility.cpp

```diff
@@ -0,0 +1,115 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ariadna Cortes
+//
+
+// Project includes
+#include "qef_utility.h"
+
+namespace Kratos {
+
+    array_1d<double,3> QEF::QEFPoint (
+        const GeometryType& rVoxel,  
+        const GeometryArrayType& rTriangles     
+    ) {
+        array_1d<double,3> x_point;
+        array_1d<double,3> center = CalculateCenter(rVoxel);
+        MatrixType m_center(3,1);
```

Comment:
```suggestion
        BoundedMatrix<double,3,1> m_center;
```
Use stack arrays whenever it's possible.

Reply:
Besides, why not using an `array_1d<double,3>`?